js/README.md

```diff
@@ -124,21 +149,26 @@ Node.js v12+ (recommended v14+)
    2. in `<ORT_ROOT>/js/common/`, run `npm ci`.
    3. in `<ORT_ROOT>/js/web/`, run `npm ci`.
 
-2. ~~Follow [instructions](https://www.onnxruntime.ai/docs/how-to/build.html#apis-and-language-bindings) for building ONNX Runtime WebAssembly. (TODO: document is not ready. we are working on it.)~~
+2. ~~Follow [instructions](https://www.onnxruntime.ai/docs/how-to/build.html#apis-and-language-bindings) for building ONNX Runtime WebAssembly. (TODO: document is not ready. we are working on it. Please see steps described as below.)~~
 
-   in `<ORT_ROOT>/`, run either of the following commands to build WebAssembly:
+   in `<ORT_ROOT>/`, run one of the following commands to build WebAssembly:
 
    ```sh
    # In windows, use 'build' to replace './build.sh'
 
    # The following command build debug.
    ./build.sh --build_wasm
 
+   # The following command build debug with debug info.
```

Comment:
Add a link to an extension, so people know how to debug with this debug info.

Reply:
it's hard to find a good place to put this link in this section. I added a new section under "debugging"